--- curl-7.59.0/include/curl/system.h.socklen_t	2018-01-30 23:02:52.000000000 +0100
+++ curl-7.59.0/include/curl/system.h	2018-04-08 17:44:33.797554000 +0200
@@ -365,7 +365,7 @@
 #    define CURL_SUFFIX_CURL_OFF_T     L
 #    define CURL_SUFFIX_CURL_OFF_TU    UL
 #  endif
-#  define CURL_TYPEOF_CURL_SOCKLEN_T socklen_t
+#  define CURL_TYPEOF_CURL_SOCKLEN_T int
 #  define CURL_PULL_SYS_TYPES_H      1
 #  define CURL_PULL_SYS_SOCKET_H     1
 
