--- curl-7.68.0/include/curl/system.h.orig	2020-01-04 22:41:58.000000000 +0000
+++ curl-7.68.0/include/curl/system.h	2020-01-30 18:17:53.804852000 +0000
@@ -396,7 +396,7 @@
 #    define CURL_SUFFIX_CURL_OFF_T     L
 #    define CURL_SUFFIX_CURL_OFF_TU    UL
 #  endif
-#  define CURL_TYPEOF_CURL_SOCKLEN_T socklen_t
+#  define CURL_TYPEOF_CURL_SOCKLEN_T int
 #  define CURL_PULL_SYS_TYPES_H      1
 #  define CURL_PULL_SYS_SOCKET_H     1
 
