--- curl-8.8.0/lib/dynhds.h.stdint	2024-05-22 07:55:00.000000000 +0000
+++ curl-8.8.0/lib/dynhds.h	2024-07-02 18:46:12.070001000 +0000
@@ -173,7 +173,6 @@
 
 #ifdef USE_NGHTTP2
 
-#include <stdint.h>
 #include <nghttp2/nghttp2.h>
 
 nghttp2_nv *Curl_dynhds_to_nva(struct dynhds *dynhds, size_t *pcount);
--- curl-8.8.0/lib/dynhds.c.stdint	2024-05-22 07:55:00.000000000 +0000
+++ curl-8.8.0/lib/dynhds.c	2024-07-02 18:47:19.800002000 +0000
@@ -28,7 +28,6 @@
 
 /* The last 3 #include files should be in this order */
 #ifdef USE_NGHTTP2
-#include <stdint.h>
 #include <nghttp2/nghttp2.h>
 #endif /* USE_NGHTTP2 */
 #include "curl_printf.h"
